src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
Add an empty line between the closing head tag and the opening body tag to add some 'air' and simplify reading, as per the checklist.